BlueprintUI/Tests/EqualStackTests.swift

```diff
@@ -61,6 +61,77 @@ class EqualStackTests: XCTestCase {
 
     }
 
+    func test_measuring_constrained() {
+
+        let children = [
+            AreaElement(area: 100),
+            AreaElement(area: 1200), // The only one affecting the cross-axis size
+            AreaElement(area: 100),
+        ]
+
+        // direction = .horizontal
+        do {
+            let constraint = SizeConstraint(width: .atMost(300), height: .unconstrained)
+
+            // spacing = 0
+            do {
+                let stack = EqualStack(direction: .horizontal) { stack in
+                    stack.spacing = 0
+                    stack.children = children
+                }
+
+                // 300 / 3 = 100
+                // 1200 / 100 = 12
+                XCTAssertEqual(stack.content.measure(in: constraint), CGSize(width: 300, height: 12))
+            }
+
+            // spacing = 50
```

Comment:
Your comment doesn't match your test value

Reply:
Ah, yep, changed it later to make the math in the comment cleaner. Will update the comment